projects/storefrontstyles/scss/components/cart/_cart-details.scss

```diff
@@ -17,4 +17,15 @@
     color: var(--cx-color-success);
     padding: 0.5rem 0;
   }
+
```

Comment:
Since this is an existing component, this might be a potential breaking change for CSS, as well as for the HTML. I need to look at this closer once I'm able to reproduce the UI.